server/config/brain/humidity-sensor/answers.fr.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "label": "humidity-sensor.get-in-room.success",
-    "answers": ["Il fait {{ humidity }} {{ unit }} dans la pièce {{ roomName }}."]
+    "answers": ["Le taux d'humidité est de {{ humidity }} {{ unit }} dans la pièce {{ roomName }}."]
```

Comment:
I think it'll output :

"Le taux d'humidité est de 76 percent dans la pièce salon".

Maybe put "%" ?